routing/message/handlers/passthrough/device_handler_test.go

```diff
@@ -54,6 +75,272 @@ func TestHandleRootFeatureEvent(t *testing.T) {
 	assert.Equal(t, `{"state":{"reported":{"status":200}}}`, messagePayload)
 }
 
+func TestUpdateRootFeaturePropertiesModify(t *testing.T) {
+	payload := `{
```

Comment:
There are many repetitions here. The new tests could be refactored a lot using either tables and/or some global variables.

Reply:
I think I managed to achieve good balance between minimizing duplication and having clear independent tests.
Even if there is some duplication in some of the payload, or expected jsons, I would say it is just coincidence. 
I would prefer to keep these definitions in the testcases and avoid coupling separate testcases with shared global variables.
Please take a look and let me know what do you think about the current state of the tests.